login.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body class="bg">
   <div>
-    <img class="logo" src="images/logo.png">
```

Comment:
why  change the logo to jpg from png? png holds transparency.

Reply:
I was trying to fix it and I started with a jpg.  I'll change it back.

Reply:
ahhh